app/webpacker/packs/components/step-by-step-nav.js

```diff
@@ -0,0 +1,476 @@
+// Adapted from https://raw.githubusercontent.com/alphagov/govuk_publishing_components/master/app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js
+
+/* eslint-disable */
```

Comment:
Hahaha, I love that we just disable linter for all of that 😂

Just to be sure... Our govuk-components don't have this particular component, right?

Reply:
No, this generally lives on a top level gov.uk page, so isn't included anywhere. Publishing components obviously has it, but I really don't want to jump through all the hoops to use that...